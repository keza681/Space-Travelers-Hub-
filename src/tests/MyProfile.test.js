import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../Redux/ConfigureStore';
import Myprofile from '../components/HomeRocket/MyProfile';

describe('Profile Component ', () => {
  it('Should renders the profile page succefully', () => {
    const app = renderer.create(
      <Provider store={store}>
        <Myprofile />
      </Provider>,
    ).toJSON();
    expect(app).toMatchSnapshot();
  });
});
