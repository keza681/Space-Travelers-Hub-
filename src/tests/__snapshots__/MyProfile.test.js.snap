// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile Component  Should renders the profile page succefully 1`] = `
<div
  className="wrapper_profile"
>
  <div
    className="profile"
  >
    <h3>
      My Rockets
    </h3>
    <ul
      className="list"
    />
  </div>
  <div
    className="profile"
  >
    <h3>
      My Missions
    </h3>
    <ul
      className="list"
    />
  </div>
</div>
`;
